crates/shared/src/gas_price_estimation/mod.rs

```diff
@@ -26,6 +27,7 @@ pub enum GasEstimatorType {
     Web3,
     BlockNative,
     Native,
+    Driver,
```

Comment:
Since this estimator type can't live without a URL, can that separate config be incorporated right here?

```suggestion
    Driver(Url),
```

That would probably require a custom parser implementation, but that would make the estimator initialization logic less awkward.

Reply:
> however, for this I want to make sure things work and we don't have to roll back

I got your point. Ok, let's keep it as is, then at least for now.